### DISCUSSION TITLE
New MicroPython UI tool (IDE & more) for MacOS/Windows/Linux.

### DISCUSSION THREAD

**Discussion by jczic** [micropython/micropython#10552] [Show and tell]
Hello everyone!

I just wanted to introduce my latest open-source software, available on GitHub: [ESP32 MPY-Jama](https://github.com/jczic/ESP32-MPY-Jama).

![ESP32 MPY-Jama](https://github.com/jczic/ESP32-MPY-Jama/raw/master/img/ESP32%20MPY-Jama%20NEW.png)

It's a UI tool that allows you to manipulate and develop the ESP32 using MicroPython.
It natively includes features such as an IDE, file management and transfer, real-time dashboard display, Wi-Fi management, and the ability to import and develop customizable mini-features.

It works on MacOS (arm64/Intel x86_64), Windows 64-bit, and Linux.
For Mac and Windows, there is also a binary installation option available.

Thank you in advance for any feedback you may provide! :)

JC`zic

**Comment by bellad17** [comment]
i've " Suspicious:W32/Malware!DeepGuard " with fsecure ????

**Comment by jczic** [comment]
Hmm! @bellad17 Do you have run binaries with the setup or sources ?
I've used InstallForge for the Windows setup and PyInstaller for the package of Python software.
Do you have details about this ?
Thank you.

**Comment by bellad17** [comment]
have run binaries with the setup
j'ai utilisé [ESP32.MPY-Jama.v1.0.3-windows.exe]
au lancement fsecure  me mets " Suspicious:W32/Malware!DeepGuard " et le deplace en quarantaine

**Reply by jczic** [reply]
Ok I see.
So I use InstallForge (https://installforge.net) for the setup.
I think either FSecure doesn't like open-source programs that install something (only one executable and 2 shortcuts), or it's only because the executable is not signed for Windows.
The .app bundle under MacOS is, on the other hand.
I don't think FSecure gives details on why it doesn't analyze it like that obviously...
Thank you for your feedback agin and for using it @bellad17 :)

(You can also try the previous release because it is directly the executable of the program, zipped, without setup : https://github.com/jczic/ESP32-MPY-Jama/releases/tag/v1.0.2)
